docs/manual/getting_started/installation/core_cpp_common_build.rst

```diff
@@ -0,0 +1,27 @@
+.. The following text is included in installation.cpp.rst and core_cpp_common.rst (at the end of the files).
+
+If you do not require the examples, use ``-DBUILD_EXAMPLES=ON`` to omit them.
+
+To install |var-project-short| after a successful build:
+
+.. code-block:: bash
+
+    cmake --build . --target install
+
+The install step copies everything to:
+
+ -  ``<install-location>/lib``
+ -  ``<install-location>/bin``
+ -  ``<install-location>/include/ddsc``
+ -  ``<install-location>/share/CycloneDDS``
+
+.. note::
+    Depending on the installation location, you may need administrator privileges. 
+
+At this point, you are ready to use Cyclone DDS in your projects.
+
+.. note:: Build types
+
+    The default build type is a release build that includes debugging information (``RelWithDebInfo``).
+    This build is suitable for applications because of its high-performance and debugging capabilities.
+    If you prefer a Debug or pure Release build, add ``-DCMAKE_BUILD_TYPE=<build-type>`` to your CMake invocation.
```

Comment:
It would be silly, but a complete stranger to this type of thing would ask the question  "What are the `<build-type>`'s available then?". We could make this highlight the flags inline, or refer to them elsewhere (if a general appendix with options available exists).

For an in-line I would imagine something like:
 ```suggestion
    If you prefer a `Debug` or pure `Release` build, add ``-DCMAKE_BUILD_TYPE=<build-type>`` to your CMake invocation.
```

Reply:
Added italics to debug and release